output/static-analysis-demo.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
No need to commit this file. We will keep only the 4 projects in the report for organization purposes. But no problem, I can delete it later

Reply:
Thanks!